indexer-app/src/cmr/indexer/data/index_set.clj

```diff
@@ -1049,13 +1050,10 @@
 (defn resolve-generic-concept-type
   "if the concept type is generic, figure out from the concept what the actual document type is"
   [concept-type concept]
-  (if (= :generic concept-type)
-    (let [reported-name (clojure.string/lower-case (get-in concept [:MetadataSpecification :Name]))
-          reported-version (get-in concept [:MetadataSpecification :Version])
-          approved (approved-generic?
-                    (keyword reported-name)
-                    reported-version)]
-      (when approved (keyword (format "generic-%s" reported-name))))
+  ;; TODO: Generic work: This let block occurs multiple times we should either figure out that we don't need 
```

Comment:
the let block has been removed so i am unsure if this comment was also intended to be removed, or just reworded bc it's actually referring to the function below. with that said i am all for DRY

Reply:
what is DRY?

Reply:
I agree.  I have removed the TODO comment.